typescript/ccip-server/src/services/CCTPService.ts

```
@@ -27,18 +29,25 @@ class CCTPService extends BaseService {
   cctpAttestationService: CCTPAttestationService;
   public readonly router: Router;
 
-  static async initialize(): Promise<BaseService> {
+  static async initialize(logger: Logger): Promise<BaseService> {
     const env = EnvSchema.parse(process.env);
     const multiProvider = await this.getMultiProvider(env.REGISTRY_URI);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)